### ISSUE TITLE
Server issues

### ISSUE THREAD

**Issue by baskettcase** [skiplagged/pokemongo-php#17]
@aktarer Is there anything that we can do? The tool has been pretty much unusable today, I get more skiplagged errors than I get pokemon. Is there any way that we can get the listings without touching skiplagged and then at that point send it on over to you? instead of the other way around? Im starting to look for other solutions and I would rather not :(

**Comment by aktarer** [comment]
Sorry, our api endpoint grew significantly in popularity. Even then uptime on our end has been over 99%. Niantic's login servers were down a lot yesterday so that was likely the issue. Though not completely accurate, you can check the status of that here: http://ispokemongodownornot.com/

**Comment by baskettcase** [comment]
I tagged all the errors with where they are coming from and I'm getting a lot of this

1469804713.786 failed to get pdata from SKIPLAGGED
1469804714.064 failed to retrieve profile from SKIPLAGGED

**Comment by aktarer** [comment]
We recently imposed a rate limit of 1 request per second due to high traffic. Try the latest code

**Comment by baskettcase** [comment]
Ok thanks Ill try the latest code, hopefully it's optimized since Im getting that 1 request per second and only running the script once.

**Comment by baskettcase** [comment]
Ok so that worked, the one sleep(1) code seems to return less errors now.

**Comment by baskettcase** [comment]
Is there a problem with PTC logins with this library right now? They always fail at this point. I even tried a different PTC account and still can't login. Ideas?

**Comment by aktarer** [comment]
Everything looks fine on our end. Check http://ispokemongodownornot.com/

**Comment by baskettcase** [comment]
I've noticed that it says everything is good there, but Im getting a bunch of "failed to get login_url will try again.", I added a limit of 5 times since it was going like crazy with a bunch of failed to get login_url's. This is in your PokemonTrainerClub->getAccessToken()

**Comment by wearelogik** [comment]
All services are up and running but still get repeated "failed to retrieve specific api endpoint"

**Comment by baskettcase** [comment]
Yep Im getting a lot of errors right now.. is your server having problems?

**Comment by baskettcase** [comment]
I took a look at your map and it is showing the ones that I am requesting, but you're not sending them back to me.. hmm... https://skiplagged.com/pokemon/#48.418,-122.284,16

**Comment by wearelogik** [comment]
Are you successfully finding pokemons with this library right now? Been scanning forever without any luck.

**Comment by baskettcase** [comment]
Yes I am, once skiplagged started working for me then it started returning pokemons again also.

**Comment by baskettcase** [comment]
Looks like Im not receiving any pdata and requests from skiplagged this morning.. any ideas when that will be working again?

**Comment by baskettcase** [comment]
Same thing this morning! Just a "for your information"! :)

**Comment by wearelogik** [comment]
This project is dead.

-----Original Message-----

**Comment by baskettcase** [comment]
Hi @aktarer is what @wearelogik says true? is it dead? I see that you pulled your android app also...
